Lib/subprocess.py

```diff
@@ -608,25 +609,15 @@ def __init__(self, args, bufsize=-1, executable=None,
         if not isinstance(bufsize, int):
             raise TypeError("bufsize must be an integer")
 
+        if close_fds is _PLATFORM_DEFAULT_CLOSE_FDS:
+            close_fds = True
```

Comment:
I suggest to now remove _PLATFORM_DEFAULT_CLOSE_FDS and always set default to True.